gradle.properties

```diff
@@ -1,3 +1,10 @@
+# Project-wide Gradle settings.
+
+org.gradle.jvmargs=-Xmx1536m
```

Comment:
When we are making these changes, it'd be great if we can take the opportunity to increase this value. We used to have it set low for most projects because CircleCI machines only had 2GB of memory. However, our current Android CIs have significantly more, so I usually set these to 6G whenever I get the chance.

I realize that this is just moving the current value, so I'll leave it to you to whether to include my suggestion in this PR or not.

Reply:
Great suggestion @oguzkocer , keep them coming! 🙇 ❤️ 🚀 

Done: a477e091424f145caf96d7e8bdc351a3b89ccb13